main.py

```
@@ -0,0 +1,70 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+from typing import Optional
```

Ruff (0.8.0)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)